hack/lib/etcd.sh

```diff
@@ -117,6 +117,10 @@ kube::etcd::install() {
     os=$(kube::util::host_os)
     arch=$(kube::util::host_arch)
 
+    if [[ ! -d "${KUBE_ROOT}/third_party" ]]; then
```

Comment:
@shamsher31 we don't want to fail silently :) we want the user to see lines 126-127

Reply:
> @shamsher31 I think the script is expected to run from k8s root dir `~/go/src/k8s.io/kubernetes/`. It always has `third_party` https://github.com/kubernetes/kubernetes/tree/master/third_party


I directly cloned https://github.com/dims/hyperkube and was trying to run `./hack/install-etcd.sh` and saw the following error.
```
$ ./hack/install-etcd.sh 
$HOME/hyperkube/hack/lib/etcd.sh: line 120: cd: $HOME/hyperkube/third_party: No such file or directory
!!! [1005 04:33:32] Call tree:
!!! [1005 04:33:32]  1: ./hack/install-etcd.sh:28 kube::etcd::install(...)
```

> @shamsher31 we don't want to fail silently :) we want the user to see lines 126-127

@dims If this check is not present then I had to manually create `third_party` folder for clean install to work. 
I assumed if `third_party` folder is not present, the script should create it and proceed to the next step instead of failing.?

Reply:
good point, could you also remove the `|| return 1` in line 124 as we are already ensuring that the directory is present?

Reply:
Make sense. 👍

Reply:
> I directly cloned https://github.com/dims/hyperkube

oops right, I forgot we are running it from hyperkube not k/k. In this case it is right to merge it here as there is no hyperkube repo upstream. Sorry for confusing you :)